sdk/src/main/java/io/radar/sdk/model/RadarInAppMessage.kt

```diff
@@ -0,0 +1,161 @@
+package io.radar.sdk.model
+
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * Data class representing the payload for an in-app message.
+ * Contains all the necessary information to display a banner message.
+ */
+data class RadarInAppMessage(
+    val title: Title,
+    val body: Body,
+    val button: Button? = null,
+    val image: Image? = null,
+    val metadata: JSONObject? = null
```

Comment:
I think theoretically metadata will always be non-null because it's always going to contain the compaign info